frontend/src/pages/Projects/Project/ChangeRequest/FormConfig.ts

```diff
@@ -81,11 +99,12 @@ export const FormConfig = (query: UseQueryResult<AxiosResponse, unknown>) => {
       required: true,
     },
     {
+      //TODO: Broken, wil be updated in future ticket
       fieldName: "types",
       fieldType: "multiText",
       fieldLabel: "Types",
       width: "full",
-      required: true,
+      // required: true,
```

Comment:
is this comment suppose to be here?

Reply:
I'll delete the comment but keep the TODO on L102